collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="/must-gather"
+NAMESPACE_PATH="namespaces/openshift-kube-apiserver"
+POD_PATH="pods/kube-apiserver/kube-apiserver"
+APF_PATH="${BASE_COLLECTION_PATH}/${NAMESPACE_PATH}/${POD_PATH}/api_priority_and_fairness"
+
+mkdir -p ${APF_PATH}
+MASTER_NODES=$(oc get endpoints -n default kubernetes -o jsonpath='{.subsets[*].addresses [*].ip}')
```

Comment:
nit: I would rename it to `KUBE_API_SERVERS`

Reply:
Done